package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ai-flow-dashboards",
+  "version": "3.0.0.0",
+  "description": "OpenSearch AI Flow Dashboards Plugin",
+  "main": "index.js",
+  "config": {
+    "plugin_version": "3.0.0.0",
+    "plugin_name": "aiFlowDashboards",
+    "plugin_zip_name": "ai-flow-dashboards"
+  },
+  "private": true,
+  "scripts": {
+    "plugin-helpers": "../../scripts/use_node ../../scripts/plugin_helpers",
+    "osd": "../../scripts/use_node ../../scripts/osd",
+    "opensearch": "node ../../scripts/opensearch",
+    "lint": "node ../../scripts/eslint .",
```

Comment:
Why do some of these use `use_node` and others don't?

Also, plugins should also have a `lint:style` script, that calls `../../scripts/use_node ../../scripts/stylelint` (which should be part of the CI workflow, along with `lint:es`)

Reply:
I'll clean up the `use_node` vs. `node` differences, I referenced some of this from different existing plugins.

Thanks for the suggestions on the lint scripts - I'll work on that + adding a github workflow for them